tests/plugins/preview/manual/previewcdn.html

```diff
@@ -0,0 +1,72 @@
+<div id="editor"></div>
+<div id="loading_box" style="margin: 20px 0"></div>
+<div id="editor_cdn">
+	<h1>Sample <span style="color: red;">document</span></h1>
+	<p>Lorem ipsum</p>
+	<div style="page-break-after:always">
+		<span style="display:none">&nbsp;</span>
+	</div>
+	<p>dolor sit amet.</p>
+</div>
+
+<script>
+	// 1. Load local editor by default with bender.
+	// 2. Remove existing instance and add script with editor from CDN.
+	// 3. Wait for the new editor to be loaded from CDN and initialize it.
+	// 4. Add icons in IE <11 from local.
+	var path = 'https://raw.githack.com/ckeditor/ckeditor4/903fa05e6bde756a7f4e618b6d24b4db21da5ab7';
+
+	CKEDITOR.replace( 'editor' );
+	CKEDITOR.once( 'instanceReady', function() {
+		var scripts = document.querySelectorAll( 'script' ),
+			head = document.head || document.getElementsByTagName("head")[0];
```

Comment:
```suggestion
			head = document.head || document.getElementsByTagName( 'head' )[ 0 ];
```

However it would be more elegant to use our `CKEDITOR.dom` methods instead of native ones.

Reply:
Good point. Thanks 👍